DataFormats/METReco/src/classes_def.xml

```diff
@@ -164,14 +164,18 @@
   </class>
   <class name="edm::Wrapper<reco::EcalHaloData>"/>
 
-  <class name="reco::HcalHaloData" ClassVersion="10">
+  <class name="reco::HcalHaloData" ClassVersion="11">
    <version ClassVersion="10" checksum="3928363167"/>
+   <version ClassVersion="11" checksum="3305429659"/>
   </class>
   <class name="edm::Wrapper<reco::HcalHaloData>"/>
 
+  <class name="std::vector<HaloTowerStrip>"/>
```

Comment:
The following is missing (see what's in 76X ) before the vector of HaloTowerStrip is defined
```diff
+  <class name="HaloTowerStrip" ClassVersion="2">
+   <version ClassVersion="2" checksum="665188928"/>
+  </class>
```